src/activeclass.jl

```diff
@@ -1,35 +1,43 @@
 module ActiveClass
 
-    using ..Lib
-    using ..Utils
+    import ..Lib
+    import ..Utils
+    import ..TypedDocStringExtensions
+
+    TypedDocStringExtensions.@template (FUNCTIONS, METHODS) =
```

Comment:
Interesting. I didn't know you could do this.

Reply:
Pretty neat stuff. I'm just discovering it as well.

This is what is looks like:

![screen shot 2019-02-07 at 10 59 24 am](https://user-images.githubusercontent.com/1813121/52432379-86cdbf00-2ac7-11e9-800d-fca8f7bc5c48.png)

Reply:
Resolved in f08087a.